scripts/pre_push_hook.py

```diff
@@ -272,11 +272,23 @@ def start_linter(files):
     return task.returncode
 
 
-def start_python_script(scriptname):
-    """Runs the 'start.py' script and returns the returncode of the task."""
+def run_script_and_get_returncode(script_name):
```

Comment:
> The / is because of the path. Example: scripts/run_frontend_tests should be scripts.run_frontend_tests. Also, I think it would better to pass the scriptname instead because it will avoid repeating the construction of python scripts everytime. What do you think?

I thought about it and I think this function is too complex. A lot of the complexity is introduced because you are taking strings and doing a bunch of manipulation with them. Passing the cmd array in directly removes all of that. So I would advocate for passing the cmd array in directly and doing no further processing on it.

Reply:
Done. I've created another function to deal with string manipulation.